dropwizard-testing/src/main/java/io/dropwizard/testing/junit5/DropwizardAppExtension.java

```diff
@@ -175,6 +179,16 @@ public void onRun(C configuration, Environment environment, DropwizardAppExtensi
         });
     }
 
+    @Override
+    public void beforeAll(ExtensionContext extensionContext) throws Exception {
+        this.before();
+    }
+
+    @Override
+    public void afterAll(ExtensionContext extensionContext) {
+        this.after();
+    }
+
```

Comment:
Wouldn't this lead to unexpected results when static and instance extensions are mixed and nested test classes are used?

Reply:
Regarding your question about static/instance extensions:
https://junit.org/junit5/docs/current/user-guide/#extensions-registration-programmatic-static-fields

Do you have any specific problems in mind?

If we want to be able to distinguish between class level and instance level usage we could provide two extensions:
* one that implements BeforeAll- and AfterAllCallback (like JUnit 4's class rules)
* one that implements BeforeEach- and AfterEachCallback (like an instance rule)